<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Image Crawling</string>
  </property>
  <widget class="QPushButton" name="pathBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>저장 경로 설정</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtWindow1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;여기에 검색 키워드를 입력해주세요.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keywordBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>검색어 입력</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtWindow3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtWindow2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>검색어 :</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
